api/python/tests/test_formats.py

```diff
@@ -3,30 +3,33 @@
 import numpy as np
 import pandas as pd
 import pytest
+from anndata import AnnData
 
 from quilt3.formats import FormatRegistry
 from quilt3.util import QuiltException
 
 # Constants
 
+data_dir = pathlib.Path(__file__).parent / 'data'
+
 
 # Code
-def test_buggy_parquet():
+@pytest.mark.parametrize('parquet_handler', FormatRegistry.for_format('parquet'))
+def test_buggy_parquet(parquet_handler):
     """
     Test that Quilt avoids crashing on bad Pandas metadata from
     old pyarrow libaries.
     """
-    path = pathlib.Path(__file__).parent
-    for parquet_handler in FormatRegistry.for_format('parquet'):
-        with open(path / 'data' / 'buggy_parquet.parquet', 'rb') as bad_parq:
-            # Make sure this doesn't crash.
-            parquet_handler.deserialize(bad_parq.read())
+    path = data_dir / 'buggy_parquet.parquet'
+    data = path.read_bytes()
+    # Make sure this doesn't crash.
+    parquet_handler.deserialize(data)
 
 
 def test_formats_for_obj():
     arr = np.ndarray(3)
 
-    fmt = FormatRegistry.for_obj(arr)[0]
+    [fmt] = FormatRegistry.for_obj(arr)
```

Comment:
Do we really need this sort of changes? Also I think that's more Python:
```suggestion
    fmt, = FormatRegistry.for_obj(arr)
```

Reply:
you’re right, those are unrelated